java/proto/server/secure_proxy.proto

```diff
@@ -0,0 +1,32 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.server.encrypted;
+
+import "oak_remote_attestation/proto/v1/service_streaming.proto";
+
+option java_multiple_files = true;
+option java_package = "com.google.oak.server.encrypted";
+
+// Secure proxy wraps a service in an encrypted connection.
+service SecureProxy {
```

Comment:
Is it not possible to import this into secure_sever.proto rather than defining it twice?